src/Sylius/Bundle/ProductBundle/Behat/ProductContext.php

```diff
@@ -292,4 +292,35 @@ public function theFollowingOptionTranslationsExist(TableNode $table)
 
         $manager->flush();
     }
+
+    /**
+     * @Given there is a disabled product :name
+     */
+    public function thereIsDisabledProduct($name)
+    {
+        $this->thereIsProduct($name, false);
+    }
+
+    /**
+     * @Given there is an enabled product :name
+     */
+    public function thereIsEnabledProduct($name)
+    {
+        $this->thereIsProduct($name, true);
+    }
+
+    /**
+     * @Given /^There is a product named "([^""]*)"$/
```

Comment:
It depends how would you like to define this step.
If it'll be ``@Given Product "..." is enabled/disable``, meaning of this sentence require product existence. Of course you can extract ``thereAreProducts`` method product creation part to separate function and use it. However, I think the first one is better approach for now, but it's just me ;)